commit 4ec7ed554d3d8634980ccf2fd900b322b4c9b508
Merge: bfd3b3c 8b1d90e
Author: Scott Lahteine <thinkyhead@users.noreply.github.com>
Date:   Wed Oct 26 21:23:34 2016 -0500

    Merge pull request #5093 from esenapaj/Follow-up-the-PR-#5087-etc
    
    Follow-up the PR #5087 (Fixed SERVO pins on Azteeg X3), etc

diff --git a/Marlin/pins_AZTEEG_X3.h b/Marlin/pins_AZTEEG_X3.h
index 63be892..26f274e 100644
--- a/Marlin/pins_AZTEEG_X3.h
+++ b/Marlin/pins_AZTEEG_X3.h
@@ -32,6 +32,9 @@
 
 #include "pins_RAMPS_13.h"
 
+//
+// Servos
+//
 #undef SERVO0_PIN
 #undef SERVO1_PIN
 #define SERVO0_PIN  44  // SERVO1 port
diff --git a/Marlin/pins_RAMPS.h b/Marlin/pins_RAMPS.h
index 00f9308..e1d9c87 100644
--- a/Marlin/pins_RAMPS.h
+++ b/Marlin/pins_RAMPS.h
@@ -344,14 +344,14 @@
       #else
         #define BTN_EN1 37
         #define BTN_EN2 35
-        #define BTN_ENC 31  // the click
+        #define BTN_ENC 31 // the click
       #endif
 
       #if ENABLED(G3D_PANEL)
         #define SD_DETECT_PIN 49
         #define KILL_PIN 41
       #else
-        //        #define SD_DETECT_PIN -1  // Ramps doesn't use this
+        //#define SD_DETECT_PIN -1 // Ramps doesn't use this
       #endif
 
     #endif
